package.json

```diff
@@ -166,36 +165,35 @@
     "postinstall-postinstall": "^2.1.0",
     "psl": "^1.9.0",
     "react": "18.2.0",
-    "react-compiler-runtime": "file:./lib/react-compiler-runtime",
     "react-dom": "^18.2.0",
     "react-image-crop": "^11.0.7",
     "react-keyed-flatten-children": "^3.0.0",
-    "react-native": "0.74.1",
-    "react-native-compressor": "^1.8.24",
-    "react-native-date-picker": "^4.4.2",
-    "react-native-drawer-layout": "^4.0.0-alpha.3",
-    "react-native-gesture-handler": "2.20.0",
+    "react-native": "0.76.1",
+    "react-native-compressor": "1.9.0",
+    "react-native-date-picker": "^5.0.7",
+    "react-native-drawer-layout": "^4.0.1",
+    "react-native-gesture-handler": "2.20.2",
     "react-native-get-random-values": "~1.11.0",
-    "react-native-image-crop-picker": "0.41.2",
+    "react-native-image-crop-picker": "0.41.4",
     "react-native-ios-context-menu": "^1.15.3",
-    "react-native-keyboard-controller": "^1.14.0",
+    "react-native-keyboard-controller": "^1.14.4",
     "react-native-mmkv": "^2.12.2",
-    "react-native-pager-view": "6.2.3",
-    "react-native-picker-select": "^9.1.3",
+    "react-native-pager-view": "6.5.0",
```

Comment:
https://github.com/bluesky-social/social-app/pull/6211#issuecomment-2486747425

Reply:
Awesome, thank you for the heads up!